samples/snippets/.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
```

Comment:
There is already an editorconfig at repo root.

If you want to enable security analyzers for this directory only, I think the `root = true` should be removed.